src/overview/repository.d

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+
+    Utilities to work fetch single repository metadata
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module overview.repository;
+
+import octod.core;
+import octod.api.repos;
+
+import semver.Version;
+
+/**
+    Describes single repository metadata
+ **/
+struct Repository
+{
+    /// project name
+    string          name;
+    /// versions tagged in the project (key is git SHA)
+    Version[string] tags;
+    /// current versions of dependencies (key is dependency name)
+    Version[string] dependencies;
+    /// is project tested with D2 in Jenkins?
+    bool            d2ready;
```

Comment:
Mmm, I see this is still too sociomantic specific. I had the idea this versioning definition / tooling was going to be generic. I think we might think a bit more how to do specific and general stuff separately. Maybe make this d2ready thing optional via config / cmd-line options too, so at least we keep it as a hack for us and for now.